tests/wpt/web-platform-tests/performance-timeline/po-observe.any.js

```diff
@@ -47,4 +46,4 @@
     observer.observe({entryTypes: ["measure"]});
     self.performance.mark("mark1");
     self.performance.measure("measure1");
-  }, "replace observer if already present");
+  }, "replace observer if already present");
```

Comment:
I don't know why it changes at line 49, 50?

Reply:
CRLF perhaps?

Reply:
It seems the origin is CRLF, and I use LF.
I try again.